yarn.lock

```diff
@@ -8403,6 +8403,14 @@ ilp-packet@^3.0.8, ilp-packet@^3.0.9:
     extensible-error "^1.0.2"
     oer-utils "^5.0.0"
 
+ilp-packet@^3.1.0-alpha.0:
```

Comment:
monorepo releases defaulting to alpha huh

Reply:
Yeah, I don't quite have the hang of lerna monorepo management. This version works fine, though.